src/sentry/search/events/datasets/discover.py

```diff
@@ -677,6 +677,109 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     default_result_type="number",
                     private=True,
                 ),
+                SnQLFunction(
+                    "spans_histogram",
+                    required_args=[
+                        SnQLStringArg("spans_op"),
+                        SnQLStringArg("spans_group"),
+                        # the bucket_size and start_offset should already be adjusted
+                        # using the multiplier before it is passed here
+                        NumberRange("bucket_size", 0, None),
+                        NumberRange("start_offset", 0, None),
+                        NumberRange("multiplier", 1, None),
+                    ],
+                    snql_column=lambda args, alias: Function(
+                        "plus",
+                        [
+                            Function(
+                                "multiply",
+                                [
+                                    Function(
+                                        "floor",
+                                        [
+                                            Function(
+                                                "divide",
+                                                [
+                                                    Function(
+                                                        "minus",
+                                                        [
+                                                            Function(
+                                                                "multiply",
+                                                                [
+                                                                    Function(
+                                                                        "arrayJoin",
+                                                                        [
+                                                                            Function(
+                                                                                "arrayFilter",
+                                                                                [
+                                                                                    Lambda(
+                                                                                        [
+                                                                                            "z",
+                                                                                            "x",
+                                                                                            "y",
```

Comment:
nit: is there a reason this needs to be z,x,y instead of x,y,z?

Reply:
oh we shuffled the args but yeah this order doesn't make sense. i've updated that 👍